File: SED-Back/src/main/java/com/sedback/SEDBack/Dtos/ResultadoDto.java
package com.sedback.SEDBack.Dtos;

import com.sedback.SEDBack.Modelo.ComportamientoPlantilla;
import com.sedback.SEDBack.Modelo.DetallePlantilla;
import lombok.Data;

@Data
public class ResultadoDto {
    private Long idDetalleEvaluacion;
    private DetallePlantilla detallePlantilla;
    private ComportamientoPlantilla comportamientoPlantillaSeleccionado;
}

File: SED-Front/src/app/HttpMensajes/resultado-dto.ts
import { ComportamientoPlantilla } from "../modelo/comportamiento-plantilla";
import { DetallePlantilla } from "../modelo/detalle-plantilla";

export class ResultadoDto{
    idDetalleEvaluacion : number;
    detallePlantilla : DetallePlantilla;
    comportamientoPlantillaSeleccionado : ComportamientoPlantilla;

    constructor(idDetalleEvaluacion : number,detallePlantilla : DetallePlantilla,comportamientoPlantillaSeleccionado : ComportamientoPlantilla){
        this.idDetalleEvaluacion = idDetalleEvaluacion;
        this.detallePlantilla = detallePlantilla;
        this.comportamientoPlantillaSeleccionado = comportamientoPlantillaSeleccionado;
    }
}